charts/tornjak/templates/server-statefulset.tpl

```diff
@@ -43,56 +42,60 @@ spec:
             - name: certs
               mountPath: /opt/spire/sample-keys
             - name: spire-server-socket
-              mountPath: /run/spire/sockets
+              mountPath: {{ .Values.spireServerSocketDir }}
               readOnly: false
-{{- if .Values.k8s_psat.remoteClusters }}
+            {{- if .Values.k8s_psat.remoteClusters }}
```

Comment:
maybe can make this `attestors.k8s_psat`

Reply:
done